README.md

```diff
@@ -1,221 +1,172 @@
 
-##### Testing
+This project consists of two main parts: The Angular (TypeScript) frontend and the Java backend.
 
-The build file is needed to tell the Frank!Runner where our frank is located. Create a file called `build.xml` and paste the following text in it:
+### Preparations
 
-```XML
-<project default="restart-frank2frank-flow">
-	<target name="restart-frank2frank-flow">
-		<basename property="project.dir" file="${basedir}"/>
-		<exec executable="../frank-runner/restart.bat" vmlauncher="false" failonerror="true">
-			<arg value="-Dproject.dir=${project.dir}"/>
-		</exec>
-	</target>
-</project>
-```
+#### Requirements
 
-##### Development
+There are some basic requirements that are needed to test or develop the Frank!Flow application. These requirements are:
 
-If you’d like to develop the Frank!Flow it would be handy to store the Frank!Flow project in a different location than your franks. To tell the Frank!Runner that our Frank!Flow project is located at a different location, we need to add an extra argument to the build.xml. This extra argument can be seen on line 6:
+##### Backend
 
-```XML
-<project default="restart-frank2frank-flow">
-	<target name="restart-frank2frank-flow">
-		<basename property="project.dir" file="${basedir}"/>
-		<exec executable="../frank-runner/restart.bat" vmlauncher="false" failonerror="true">
-			<arg value="-Dproject.dir=${project.dir}"/>
-			<arg value="-Dfrank-flow.frontend.path=C:\PATH_TO_YOUR_FRANK-FLOW_PROJECT\frank-flow\src\main\resources\frontend"/>
-		</exec>
-	</target>
-</project>
-```
+- [Java](https://www.java.com/nl/download/)
+- [Maven](http://maven.apache.org/)
 
-You should edit `PATH_TO_YOUR_FRANK-FLOW_PROJECT` to the correct location. As you can see, the rest of the path contains the location to the frontend distribution folder.
+##### Frontend
 
-#### Restart
+- [Node.js](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/getting-started/install)
 
-The last file is optional, because it is up to you how you restart your Frank!Runner. Some people like to use the build in Task Manager of there IDE, which will have a restart task because of the build.xml.
+#### DevTools
 
-You could also create an additional `restart.bat` so it can be called via the command line interface (or double click). This file should contain:
+There are some tools that have to be used during development. These tools are needed for linters that keep the code clean and formatted the same way. The tools get run automatically by a Git hook.
 
-```bash
-call ..\frank-runner\ant.bat
-if %errorlevel% equ 0 goto end
-rem https://superuser.com/questions/527898/how-to-pause-only-if-executing-in-a-new-window
-set arg0=%0
-if [%arg0:~2,1%]==[:] if not [%TERM_PROGRAM%] == [vscode] pause
-:end
-```
+To install the tools go to the folder `frank-flow/src/frontend` this is needed because of the way this mono repo has been structured. This might change in the feature. Run `yarn` in this folder and everything should be good to go!
```

Comment:
You forgot to end a sentence here. I also recommend you to rearrange the sentences. I propose the following text:

To install the tools go to the folder `frank-flow/src/frontend`. Run `yarn` in this folder and everything should be good to go. This is needed because of the way this mono repo has been structured. This structure might change in the feature.

Reply:
Thank you, that's a good proposal.